### PR TITLE
Add search bar for Python docs

### PR DESCRIPTION
**Pull request by amaan-n-j**
### Purpose

Add a search bar to the Python docs that allows users to search the documents

### Remarks

PR introduces (a) breaking change(s): no
PR introduces (a) deprecation(s): no

### CODE REVIEW

File: streampipes-client-python/mkdocs.yml [hunk 1]
```diff
@@ -79,6 +79,10 @@ plugins:
       version_selector: true
       css_dir: css
       javascript_dir: js
+  - search:
+      lang:
+        - en
+      seperator: '[\s\-,:_!=\[\]()"/]+'
```
Comment (bossenti) [inline streampipes-client-python/mkdocs.yml:85]: Why did you only special chars as separator?
We are on version 8 of mkdocs-material so the default seperator is `'[\s\-\.]+'`
What's your idea about changing this? :)

**Comment by bossenti** [comment]
@amaan-n-j thanks for your contribution 🙏🏼

File: streampipes-client-python/mkdocs.yml (hunk 1 above)
Comment (amaan-n-j) [reply streampipes-client-python/mkdocs.yml:85]: I thought user might not know the exact thing to search, may end up adding some special character. If you want to i can remove them and keep the defaults.

Comment (bossenti) [reply streampipes-client-python/mkdocs.yml:85]: Sounds valid
Let's try it like this

**Review by bossenti — APPROVED** [review]

**Pull request merged**